.dockerignore

```diff
@@ -0,0 +1,5 @@
+target
+Dockerfile
+.dockerignore
+.git
+.gitignore
```

Comment:
To keep a consistent style with the other files, add a line at the end of the file 🙂